src/modules/user/auth.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
@gerimujo496 Not enough test coverage, we need several test cases to cover all authentication functionalities